packages/engine-server/src/enginev2.ts

```diff
@@ -505,6 +505,11 @@ export class DendronEngineV2 implements DEngine {
       onlyDirectChildren,
       originalQS,
     });
+
+    if (items.length === 0) {
```

Comment:
I think this is the correct fix / behavior - but I wonder if this may cause any regressions in existing code that calls this API and expects a `500` kind of error / exception to be happening.

Reply:
i don't think it does. as long as we have a test to catch further regressions, this looks good

Reply:
Looked through all the use cases and aside from `slice.ts`, this shouldn't break.